daemon/api.go

```diff
@@ -239,12 +239,13 @@ func sysInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 	}
 
 	m := map[string]interface{}{
-		"series":     release.Series,
-		"version":    c.d.Version,
-		"os-release": release.ReleaseInfo,
-		"on-classic": release.OnClassic,
-		"managed":    len(users) > 0,
-
+		"series":         release.Series,
+		"version":        c.d.Version,
+		"os-release":     release.ReleaseInfo,
+		"on-classic":     release.OnClassic,
+		"managed":        len(users) > 0,
+		"snap-mount-dir": dirs.SnapMountDir,
```

Comment:
I think what @niemeyer wanted to see here is more:

```
[...]
"managed":        len(users) > 0,
"locations":         map[string]interace{}{
  "snap-mount-dir": dirs.SnapMountDir,
  "snap-bin-dir": dirs.SnapBinariesDir,
},
[...]
```

@niemeyer Correct me if I am wrong.